src/site/content/en/blog/css-web-vitals/index.md

```diff
@@ -0,0 +1,484 @@
+contains the font declaration for Montserrat and Roboto. These `<link>` tags
+should be placed as early in the `<head>` as possible.
+
+
+{% Aside %}
+
+To only load a subset of a font from Google Fonts, add the
+[`?text=`](https://developers.google.com/fonts/docs/getting_started) API
+parameter. For example, `?text=ABC` loads only the characters necessary to
+render "ABC". This is a good way to reduce the file size of a font.
+
+{% endAside %}
+
+
+## Animations
+
+The primary way that animations affect Web Vitals is when they cause layout
+shifts. There are two types of animations that you should avoid using:
+[animations that trigger layout](https://web.dev/animations-guide/#triggers) and
+"animation-like" effects that move page elements. Typically these animations can
+be replaced with more performant equivalents by using CSS properties like
+[`transform`](https://developer.mozilla.org/en-US/docs/Web/CSS/transform),
+[`opacity`](https://developer.mozilla.org/en-US/docs/Web/CSS/opacity), and
+[`filter`](https://developer.mozilla.org/en-US/docs/Web/CSS/filter). For more
+information, see [How to create high-performance CSS
+animations](https://web.dev/animations/).
+
+
+### Identify
+
+The Lighthouse "Avoid non-composited animations" audit may be helpful for
+identifying non-performant animations.
+
+
+
+{% Img
+    src="image/j2RDdG43oidUy6AL6LovThjeX9c2/mXgypW9x3qgvmWDLbIZx.png",
+    alt="Lighthouse's 'Avoid non-composited animations' audit",
+    width="512",
+    height="132"
+%}
+
+
+{% Aside 'caution' %}
+
+The Lighthouse "Avoid non-composited animations" audit only identifies
+non-performant _CSS animations_; JavaScript-driven animations (for example,
+using
+[`setInterval()`](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/setInterval)
+to "animate" an element) are bad for performance but will not be flagged by this
+audit.
+
+{% endAside %}
+
+
+### Fix
+
+Change the `slideIn` animation sequence to use `transform: translateX()` rather
+than transitioning the`margin-left` property.
+
+Before:
+
+```css
+.header {
+  animation: slideIn 1s 1 ease;
+}
+
+@keyframes slideIn {
+  from {
+    margin-left: -100%;
+  }
+  to {
+    margin-left: 0;
+  }
+}
+```
+
+After:
+
+```css
+.header {
+  animation: slideIn 1s 1 ease;
+}
+
+@keyframes slideIn {
+  from {
+    transform: translateX(-100%);
+  }
+
+  to {
+    transform: translateX(0);
+  }
+}
+```
+
+
+## Critical CSS
+
+Stylesheets are render-blocking. This means that the browser encounters a
+stylesheet, it will stop downloading other resources until the browser has
+downloaded and parsed the stylesheet. This may delay LCP. To improve
+performance, consider [removing unused
+CSS](https://css-tricks.com/how-do-you-remove-unused-css-from-a-site/),
+[inlining critical CSS](https://web.dev/extract-critical-css/), and [deferring
+non-critical CSS](https://web.dev/defer-non-critical-css/#optimize).
```

Comment:
```suggestion
Stylesheets are render-blocking. This means that the browser encounters a
stylesheet, it will stop downloading other resources until the browser has
downloaded and parsed the stylesheet. This may delay LCP. To improve
performance, consider [removing unused
CSS](https://css-tricks.com/how-do-you-remove-unused-css-from-a-site/),
[inlining critical CSS](/extract-critical-css/), and [deferring
non-critical CSS](/defer-non-critical-css/#optimize).
```